accounting/invoice/tasks.py

```diff
@@ -49,7 +49,7 @@ def send_invoice_prep_notification():
     template = get_template(constants.INVOICE_NOTIFICATION_TEMPLATE)
     for client_username in settings.BILLING_CYCLE_USERS:
         client = Account.objects.get(user__username=client_username)
-        provider_emails = [rate.provider.user.email for rate in client.client_hourly_rates.all()]
+        provider_emails = [rate.provider.user.email for rate in client.client_hourly_rates.all() if rate.active]
```

Comment:
@UmanShahzad Instead of fetching `all` rates from the DB, this could use `client.client_hourly_rates.filter(active=True)`.

Reply:
@itsjeyd Argh, good point. :p